schemas/remote_configuration_v1.schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$id": "remote_configuration_v1.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "description": "An object containing the set of rules used by the conditional gatherer.",
+  "descriptions": "A data structure used by the Insights Operator Gathering Conditions Service v1 API to return the remote configurations for the conditional gathering.",
+  "examples": [
+    {
+      "rules": [
+        {
+          "conditions": [
+            {
+              "alert": {
+                "name": "APIRemovedInNextEUSReleaseInUse"
+              },
+              "type": "alert_is_firing"
+            }
+          ],
+          "gathering_functions": {
+            "api_request_counts_of_resource_from_alert": {
+              "alert_name": "APIRemovedInNextEUSReleaseInUse"
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "properties": {
+    "rules": {
+      "$ref": "gathering_rules.schema.json"
+    },
+    "version": {
+      "type": "string"
+    }
+  },
+  "required": [
+    "rules"
```

Comment:
I am not sure if anything really depends on the `version` in v1 responses, but I would still declare it as a required attribute for consistency. The build tool will always include it and we want to use the schema to double-check that it really does.